packages/testing-helpers/test/helpers.test.js

```diff
@@ -68,4 +69,18 @@ describe('Helpers', () => {
     expect(el.blurCount).to.be.within(0, 1);
     expect(el !== document.activeElement).to.be.true;
   });
+
+  it('provides waitUntil to await async state', async () => {
+    let foo = false;
+    let resolved = false;
+    waitUntil(() => foo).then(() => {
+      resolved = true;
+    });
+
+    await aTimeout(100);
+    foo = true;
+    await aTimeout(100);
+
+    expect(resolved).to.be.true;
```

Comment:
As a practice question, how do you feel about `expect`ing the test to fail to begin with in order to confirm that the change actually occurred? Not a blocker in any way, just good knowledge for me to gather!

Reply:
Yeah good idea to add in an in-between check